packages/compass-components/src/components/combobox/Combobox.tsx

```diff
@@ -189,7 +192,7 @@ export function Combobox<M extends boolean>({
    */
   const isMultiselect = useCallback(
     <T extends string>(val?: Array<T> | T | null): val is Array<T> => {
-      if (multiselect && (typeof val == 'string' || typeof val == 'number')) {
+      if (multiselect && (typeof val === 'string' || typeof val === 'number')) {
```

Comment:
I changed comparisons to strings like this to `===` but left comparisons to `null` in place and just ignored it. Hard to tell at scale what people meant with each one.